app/scripts/filters/resources.js

```diff
@@ -320,7 +320,7 @@ angular.module('openshiftConsole')
         return routeWebURLFilter(route, host);
       }
       var label = (host || routeHostFilter(route));
```

Comment:
I think I won't rather display the path, since the hostname is missing anyway.. 
It could be confusing to show `<unknown host>/path/to/something`

@spadgett @jwforres thoughts ?